Formula/node_exporter.rb

```diff
@@ -0,0 +1,65 @@
+class NodeExporter < Formula
+  desc "Prometheus exporter for machine metrics"
+  homepage "https://prometheus.io/"
+  url "https://github.com/prometheus/node_exporter/archive/v0.15.2.tar.gz"
+  sha256 "940b850376f94580e88b5e99926d92899975d5792d05709cbd9c48dab0a848ad"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    mkdir_p buildpath/"src/github.com/prometheus"
+    ln_sf buildpath, buildpath/"src/github.com/prometheus/node_exporter"
+
+    system "make", "build"
+    bin.install %w[node_exporter]
```

Comment:
Nothing to install other than the binary? No doc, man page?

Reply:
Nope, not really.

There's the README on the project page (https://github.com/prometheus/node_exporter) and a single document describing how to monitor NTP (https://github.com/prometheus/node_exporter/blob/master/docs/TIME.md), but that's about it.

Reply:
Ex. as I can read the ports-file, FreeBSD doesn't install anything but the binary and startup-script: https://github.com/freebsd/freebsd-ports/blob/ccddb85648b61731933d6085b2a1e9f8446f62e5/sysutils/node_exporter/Makefile